### DISCUSSION TITLE
Розподіл завдань проекту

### DISCUSSION THREAD

**Discussion by kolgenpav** [ESEI-ZNU/SortingAlgorithms#5] [General]
Початок

**Comment by OzzeY72** [comment]
Доброго дня!
Пропоную зробити розподілення обов'язків серед розробників:
Розробник 1: Вруцький Максим
Розробник 2: Фролова Ірина
Розробник 3: Качан Дмитро
Розробник 4: Павленко Анастасія
Розробник 5: Зайцев Денис
Чекаю ваші відповіді!

**Reply by DmitriyKachan** [reply]
Добра добра, погоджуюсь
